### DISCUSSION TITLE
Logged in successfully, however no devices or entities

### DISCUSSION THREAD

**Discussion by MitchBaker** [widewing/ha-toyota-na#83] [Q&A]
I can enter my credentials, enter 2fa code, and it initializes and reports success.

However I have no devices, or entities listed.  I have a 2023 Corolla that reports Tire pressure, odometer, and Fuel level to the app on my phone.

What am I missing?

**Comment by MujtabaJafri** [comment]
I have the same issue, it may have something to do with the car model maybe... My Highlander doesn't show up even though the app shows it and I can access remote features from the mobile app, not sure if I'm missing anything either

**Comment by HMGOMEZ77** [comment]
I'm Having the same issue.. It was working fine with the old integration 2.4.10 I believe..After the update I didn't have any entities
